examples/atari_deepRL_saliency.ipynb

```diff
@@ -83,6 +83,14 @@
      "text": [
       "Action space: ['NOOP', 'FIRE', 'RIGHT', 'LEFT']\n"
      ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "A.L.E: Arcade Learning Environment (version +a54a328)\n",
+      "[Powered by Stella]\n"
```

Comment:
I noticed that this new output was added- are you still using the pinned version of `open-gym` (v. 0.19.0)? I ran into some issues with installing newer versions of `open-gym`, as I believe they updated how the ATARI games were provided with the installation in more recent versions. As a result, it looks like the generated frames/videos are also different in this update compared to those in the original notebook. I purposefully chose frames/clips (taking advantage of the determinism of setting a seed) that highlighted the "tunneling" strategy of the agent.

Reply:
Yeah, I was using 0.20.0. Fixed.